data/mods/more_classes_scenarios/cs_classes.json

```diff
@@ -260,5 +260,219 @@
+                "both": [
+                    "tshirt",
+                    "socks",
+                    "slingpack",
+                    "sneakers",
+                    "jeans",
+                    "jacket_light",
+                    "knit_scarf",
+                    "juice",
+                    "sandwich_pbj",
+                    "fairy_tales"
+                ],
+                "male": [
+                    "briefs"
+                ],
+                "female": [
+                    "panties"
+                ]
+            }
+    },
+    {
+        "type": "profession",
+        "ident": "goalie",
+        "name": "Hockey Player",
+        "description": "You were a minor-league hockey goalie before the rest of your team became zombies.  It's just you and your hockey equipment versus the undead, but at least you can cross-check them now.",
+        "points": 0,
+        "traits": [
+            "PROF_SKATER"
+        ],
+        "items": {
+            "both": [
+                "mask_hockey",
+                "hockey_stick",
+                "jersey",
+                "pants",
+                "socks",
+                "sneakers",
+                "mouthpiece",
+                "puck"
+            ],
+            "male": [
+                "briefs"
+            ],
+            "female": [
+                "sports_bra",
+                "panties"
+            ]
+        }
+    },
+    {
+        "type": "profession",
+        "ident": "batter",
+        "name": "Baseball Player",
+        "description": "You were a batter on a local minor league team before the cataclysm.  You escaped with your equipment, but how long can you survive until your innings are up?",
+        "points": 0,
+        "items": {
+            "both": [
+                "bat",
+                "helmet_ball",
+                "dress_shirt",
+                "pants",
+                "socks",
+                "cleats",
+                "mouthpiece",
+                "baseball"
+            ],
+            "male": [
+                "briefs"
+            ],
+            "female": [
+                "sports_bra",
+                "panties"
+            ]
+        }
+    },
+    {
+        "type": "profession",
+        "ident": "linebacker",
+        "name": "Football Player",
+        "description": "You were the star player for the local football team, adored by teammates and fans alike.  Now they just adore your brain.  You've still got your bulky football gear on.",
+        "points": 0,
+        "items": {
+            "both": [
+                "football_armor",
+                "tank_top",
+                "helmet_football",
+                "pants",
+                "socks",
+                "cleats",
+                "mouthpiece",
+                "football"
+            ],
+            "male": [
+                "briefs"
+            ],
+            "female": [
+                "sports_bra",
+                "panties"
+            ]
+        }
```

Comment:
Now that multi pool is the default mode, this dude could use some extra bonuses or -1 cost.
Survivor starts with warm clothing, knife, fire and water, football player only has bad armor.